### DISCUSSION TITLE
Simple way to credit some ERC20 token on a Ganache account ?

### DISCUSSION THREAD

**Discussion by svax974** [ConsenSys-archive/truffle#5341] [General Q&A]
Could someone tell me how I could "credit" one Ganache account with some ERC20 tokens ?
I may have to deploy a contract... but what next ?

Thanks !

**Comment by cliffoo** [comment]
Credit as in transferring tokens to an account?

Take a look at [`web3.eth.sendTransaction`](https://web3js.readthedocs.io/en/v1.2.11/web3-eth.html#sendtransaction), and a quick google search "Send ether with web3" will give you info.

Alternatively since you're using Ganache already, checkout [`evm_setAccountBalance`](https://trufflesuite.github.io/ganache/#evm_setAccountBalance).

**Reply by svax974** [reply]
Thanks for the reply, I found a way to accomplish what I wanted.
In fact, "credit" was more like "magically generate a balance" of some ERC20 token.
I finally just wrote a simple ERC20 compliant contract with openzeppelin base and minted tokens in the contructor so my wallet directly had some of that token upon deployment.
Regards.

**Reply by cliffoo** [reply]
Ah my mistake, I somehow completely overlooked all occurrences of the string "ERC20" in your question.

Glad you found what you needed!
